esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3417,18 +3499,56 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     if (!req.getCacheHint_isNull())
         cacheHint = req.getCacheHint();
 
+    unsigned cacheFrom, cacheTo;
+    Owned<IterateFilesCacheElement> cachedITFS = iterateFilesCache->lookup(context, cacheHint);
```

Comment:
I don't think this new cache (of from/to/total) is necessary or really works.
Not really sure what it's trying to achieve, apart from possibly caching the total.
But that would be better added to the existing caching mechanism.